google-cloud-firestore/OVERVIEW.md

```diff
@@ -490,4 +490,4 @@ the entire collection or sub-collection.
 ## Additional information
 
 Google Firestore can be configured to use gRPC's logging. To learn more, see the
-{file:LOGGING.md Logging guide}.
+[Logging guide](file:LOGGING.md).
```

Comment:
Whoops, left the "file:" there.

And actually it looks like there are a bunch of cases of this. (Maybe all for LOGGING.md?) Please fix all of them.

Reply:
Good catch! And this is why we have code reviews haha